emission/individual_tests/TestNominatim.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import unittest
+import unittest.mock as mock
+# import overpy
+import os
+from emission.core.wrapper.trip_old import Coordinate
+import requests
+import emission.core.wrapper.entry as ecwe
+import emission.core.wrapper.wrapperbase as ecww
+import emission.net.ext_service.geocoder.nominatim as eco
+import emission.analysis.intake.cleaning.clean_and_resample as clean
+
+#temporarily sets NOMINATIM_QUERY_URL to the environment variable for testing.
+NOMINATIM_QUERY_URL_env = os.environ.get("NOMINATIM_QUERY_URL")
+
+NOMINATIM_QUERY_URL = (
+    NOMINATIM_QUERY_URL_env if NOMINATIM_QUERY_URL_env is not None 
+    else eco.NOMINATIM_QUERY_URL
+)
+
+GFBK = os.environ.get("GFBK_KEY")
+
+GEOFABRIK_QUERY_URL = (
+    "https://geocoding.geofabrik.de/{}".format(GFBK) if GFBK is not None
+    else print("No key available")
+)
+
+#Creates a fake place in Rhode Island to use for testing.
+fake_id = "rhodeislander"
+key = "segmentation/raw_place"
+write_ts = 1694344333
+data = {'source': 'FakeTripGenerator','location': {'type': 'Point', 'coordinates': [-71.4128343, 41.8239891]}}
+fake_place = ecwe.Entry.create_fake_entry(fake_id, key, data, write_ts)
+
```

Comment:
please check other test cases for proper test formatting. This should be in `setup`

Reply:
this has not yet been addressed

Reply:
Assuming my workflow will eventually be used to run all of the integration tests, I've created a new shell script called `setup_integrationtests.sh` which is called in start_integration_tests.sh. The new shell script replaces setup_tests.sh's functionality, and runs createfakeplace.py. I could have just put createfakeplace.py into the `setup` directory and run it with import, but I think it is better to see during setup if if creating the fake place works. Also, creating the new script is more consistent with the general workflow I've seen in the server code. Finally, it opens the door to calling other scripts that might be necessary for other integration tests to work. 

These are my justifications, and either way it's better than creating the fake place in the test.